### ISSUE TITLE
[playground] fallback to proxy when remote server doesn't allow CORS

### ISSUE THREAD

**Issue by elf-pavlik** [json-ld/json-ld.org#364]
I try to use files hosted on github inside of playground, it fais since gh doens't set proper CORS headers. I suggest here adding fallback to use CORS proxy.

**Comment by elf-pavlik** [comment]
@dlongley maybe https://www.npmjs.com/package/cors-anywhere would do?

**Comment by dlongley** [comment]
@elf-pavlik, yeah, that may be something we could hook up. We could have the playground's JSON-LD document loader try to fetch docs normally and when failure occurs use that proxy and leave a warning on the page with CORS help to instruct people how to enable it on their server.

**Comment by elf-pavlik** [comment]
:+1: as you have it in #366

**Comment by elf-pavlik** [comment]
@l00mi posted to @rdfjs list
>Hi RDFjs and Git(hub) users,

>One of the actions defined in the Telco (which you will see in the protocol) was to find a better possibility to fetch RDF data directly from GitHub. Since yesterday you can get the major RDF formats with correct mime-type and also cors allow-all headers from rawgit.com.

>Be aware this should be used for testing only!

>e.g. https://rawgit.com/zazukoians/tbbt-ld/master/data/person/leonard-hofstadter.ttl

>Thanks to Ryan Grove ( https://rawgit.com/ ) for taking this in! You might tweet him a “thanks” @rawgit.

https://lists.w3.org/Archives/Public/public-rdfjs/2015Mar/0014.html

rawgit.com also set's correct CORS headers, maybe we can use it as fallback for contexts hosted on github?
